table/format.h

```diff
@@ -228,19 +228,20 @@ extern Status ReadBlockContents(
 // free this buffer.
 // For description of compress_format_version and possible values, see
 // util/compression.h
-extern Status UncompressBlockContents(const char* data, size_t n,
+extern Status UncompressBlockContents(const UncompressionContext& uncompression_ctx,
+                                      const char* data, size_t n,
                                       BlockContents* contents,
                                       uint32_t compress_format_version,
-                                      const Slice& compression_dict,
-                                      const ImmutableCFOptions& ioptions);
+                                      const ImmutableCFOptions &ioptions);
 
 // This is an extension to UncompressBlockContents that accepts
 // a specific compression type. This is used by un-wrapped blocks
 // with no compression header.
 extern Status UncompressBlockContentsForCompressionType(
+    const UncompressionContext& uncompression_ctx,
     const char* data, size_t n, BlockContents* contents,
-    uint32_t compress_format_version, const Slice& compression_dict,
-    CompressionType compression_type, const ImmutableCFOptions& ioptions);
+    uint32_t compress_format_version,
+    const ImmutableCFOptions &ioptions);
```

Comment:
nitpick: the '&' should stick to the type not variable?

Reply:
@miasantreble True, will fix it, not sure how this came about.